### DISCUSSION TITLE
How do I revoke my signature to vote?

### DISCUSSION THREAD

**Discussion by ghotoman** [snapshot-labs/snapshot#3345] [Q&A] [answered]
I got to a scam project page and signed votes, how can I cancel them?

**Comment by metrox-eth** [comment, marked as answer]
You've voted on a flagged space, see details here: https://docs.snapshot.org/spaces/badges-and-warnings
The largest part of flagged spaces are flagged for IP infringement reason. We don't have any evidence of scam for any space so far.
**There is no way to revoke a vote on Snapshot.**

**Reply by ghotoman** [reply]
> You've voted on a flagged space, see details here: https://docs.snapshot.org/spaces/badges-and-warnings The largest part of flagged spaces are flagged for IP infringement reason. We don't have any evidence of scam for any space so far. **There is no way to revoke a vote on Snapshot.**

ok, thx
